components/lib/splitbutton/SplitButton.js

```diff
@@ -32,6 +33,9 @@ export const SplitButton = React.memo(
         const { ptm, cx, isUnstyled } = SplitButtonBase.setMetaData(metaData);
 
         useHandleStyle(SplitButtonBase.css.styles, isUnstyled, { name: 'splitbutton' });
+        useOnEscapeKey(overlayRef, props.closeOnEscape, (event) => {
```

Comment:
Does not have `props.closeOnEscape` it should just be on.

Reply:
look at this PR for guidance: https://github.com/primefaces/primereact/pull/5361

Reply:
Oh, Okay

Reply:
Done, thanks.